doc/rest-api.md

```diff
@@ -2285,19 +2285,33 @@ Input:
         "type": "custom"
     }
 
+Input:
+
+    {
+        "config": {},
+        "pool": "pool1",
+        "name": "vol1",
+        "type": "custom"
+        "source": {
+            "pool": "pool2",
+            "name": "vol2",
+            "type": "custom"
+        }
+    }
 
 ## `/1.0/storage-pools/<pool>/volumes/<type>/<name>`
 ### POST
  * Description: rename a storage volume on a given storage pool
  * Introduced: with API extension `storage_api_volume_rename`
  * Authentication: trusted
- * Operation: sync
+ * Operation: sync/async
```

Comment:
Should be:

    Operation: sync or async (when moving to a different pool)

To clarify.

Reply:
added